frontend/src/queries/nodes/InsightQuery/utils/filtersToQueryNode.ts

```diff
@@ -474,6 +475,16 @@ export const pathsFilterToQuery = (filters: Partial<PathsFilterType>): PathsFilt
         pathGroupings: filters.path_groupings,
         funnelPaths: filters.funnel_paths,
         funnelFilter: filters.funnel_filter,
+        funnelActorsQuery: filters.funnel_filter
+            ? ({
+                  kind: NodeKind.FunnelsActorsQuery,
+                  source: {
+                      funnelsFilter: funnelsFilterToQuery(filters.funnel_filter),
+                      ...filtersToQueryNode(filters.funnel_filter),
+                  },
```

Comment:
```suggestion
                  source: filtersToQueryNode(filters.funnel_filter),
```
`filtersToQueryNode` already builds the `funnelsFilter` as well. Thus we should be able to simplify to this suggestion unless I'm missing something.